docs/how-to-guides/integrating-autoware-with-your-vehicle.md

```diff
@@ -24,6 +24,7 @@ Some examples are:
 - [HDL](https://github.com/koide3/hdl_graph_slam)
 - [LIO-SAM](https://github.com/TixiaoShan/LIO-SAM)
 - [LiDAR-SLAM (ROS 2)](https://github.com/rsasaki0109/lidarslam_ros2)
+  - You need to relay the autoware topic name so that LiDAR-SLAM can receive it.
```

Comment:
@sfukuta Could you clarify this sentence a bit more? I couldn't understand well.
I guess it's better to write some console commands instead.
cc @kminoda @shmpwk @soblin

Reply:
@kenji-miyake 
Thanks you.This sentence mean as follows.

To use LiDAR-SLAM(ROS2), I need "/tf" and "/input_cloud" as input topics.
However, "/input_cloud" is remapped to "/velodyne_points".
Autoware doesn't have "/velodyne_points" so you need to convert topics of the same type.

The topic name convert command is `ros2 run topic_tools relay <autoware topic name> /velodyne_points`

Reply:
@kminoda Would it be possible for you to give me some proposals for this change?

Reply:
Maybe something like this? Let me know if I misunderstood your thought.


> Note that if you are using the sensing modules in Autoware when creating the pointcloud map using the above tools, make sure that the topic names are properly renamed.

Reply:
But it seems to me that this side note is not about Autoware but rather about ROS 2 in general. Maybe the Autoware Documentation is not the right place to feedback the issue. 

Assuming that @.sfukuta-san had encountered above issue, how about posting that issue on Autoware Discussions?

Reply:
For example like this:

Q: I am having an issue using LiDAR-SLAM ROS2 with Autoware sensing modules. I want to create pointcloud map but the SLAM doesn't start.
A: Changing the name of output sensor topics from Autoware fixed the issue. We need to rename the topics in the following procedure...

Reply:
@kminoda 
Thanks you.I'll try posting this on the Autoware Discussions.
In that case, can I post it in the Q&A category and answer it myself?

Reply:
I guess it's OK as long as the issue and the solution is clear enough.